### PR TITLE
Add edit mode title

### LINKED ISSUE #1436: [Quest/eCBIS] - On Edit,  Household details Village Town is empty
**Describe the bug**
There is missing data for a family village or town in family details profile view, wrong labels on edit family details 

**Acceptance criteria**

- [x] 1. Click on a family from the family register 
- [x] 2. Click on the menu items 
- [x] 3. Select on the menu - Family details 
- [x] 4. The title of the page should be Edit Family details 
- [x] 5. All fields should be populated i.e Famil name, Household ID, and Village or Town
- [x] 6. Change the button text from Add family to Edit family 
- [x] 7. On save update family details 

**Screenshots**

### PR DESCRIPTION
**Pull request by FikriMilano**
**IMPORTANT: Where possible all PRs must be linked to a Github issue**

Fixes #1436 

**Checklist**
- [x] I have written **Unit tests** for any new feature(s) and edge cases for bug fixes
- [x] I have added any strings visible on UI components to the `strings.xml` file
- [x] I have updated the  [CHANGELOG.md](./CHANGELOG.md) file for any notable changes to the codebase
- [x] I have run `./gradlew spotlessApply` and `./gradlew spotlessCheck` to check my code follows the project's style guide
- [x] I have built and run the fhircore app to verify my change fixes the issue and/or does not break the app

### CODE REVIEW

**Review by pld — COMMENTED** [review]
let's add/refine tests around this

**Comment by FikriMilano** [comment]
Yeah, let me do that

File: android/engine/src/test/java/org/smartregister/fhircore/engine/ui/questionnaire/QuestionnaireActivityTest.kt [hunk 1]
```diff
@@ -441,6 +441,39 @@ class QuestionnaireActivityTest : ActivityRobolectricTest() {
...
 
+  @Test
+  fun testQuestionnaireTypeEditShouldAppendEditPrefixInActionBarTitle() {
+    questionnaireActivity.questionnaireType = QuestionnaireType.EDIT
+    questionnaireActivity.questionnaireViewModel.questionnaireConfig =
+      QuestionnaireConfig("form", "title", "form-id")
+
+    questionnaireActivity.updateViews()
+
+    Assert.assertEquals("Edit title", questionnaireActivity.supportActionBar?.title)
```
Comment (pld) [inline android/engine/src/test/java/org/smartregister/fhircore/engine/ui/questionnaire/QuestionnaireActivityTest.kt:452]: can you fetch the lhs using `getString`?

**Review by pld — CHANGES_REQUESTED** [review]

File: android/engine/src/test/java/org/smartregister/fhircore/engine/ui/questionnaire/QuestionnaireActivityTest.kt (hunk 1 above)
Comment (FikriMilano) [reply android/engine/src/test/java/org/smartregister/fhircore/engine/ui/questionnaire/QuestionnaireActivityTest.kt:452]: Hmm, could you elaborate?

**Comment by pld** [comment]
You know how in the class you're fetching the during from the text strings file with I think the getString function? Can you do the same her in the left hand side of this assert, so if we change the text in the strings file this test still passes

**Comment by FikriMilano** [comment]
Ah I see, got it

**Review by pld — APPROVED** [review]

**Pull request merged**